docs/HardWonWisdom.md

```diff
@@ -99,3 +99,16 @@ A couple more observations:
 
 So, either by mimicking the approach of the existing `ValidityInterval` validation logic or by altering the STS innards to allow short-circuiting failure, we could reify the `EpochInfo` failures into the `PredicateFailure` hierarchy, and thereby leverage the types to force each invocation of the Ledger API to independently handle the possibility of `PastHorizonException`s.
 But it's not obvious that that is definitely worth the extra complexity it would introduce on the Ledger code.
+
+## Why use the Honest Chain Growth window as the Ledger's Stability Window?
+
+Suppose we have selected a different chain than our peer and that our selected chain has L blocks after the intersection and their selected chain has R blocks after the intersection.
```

Comment:
Minor: I'd wrap up lines to 80 chars to minimize diffs and help readability.

Reply:
For Markdown files, I favor writing exactly one sentence per line in the file. It makes diffs drastically more legible.

It does harm readability, but I assume that people prioritizing readability can easily use a Markdown renderer when reading the document.

Seem reasonable?

Reply:
Fair enough. The donwside is that reading the diff on GitHub is a bit harder.